packages/lib/sdk/src/configuration/getEvidenceConfig.legacy.js

```diff
@@ -6,14 +6,19 @@ import { EvidenceConfigSchema } from './schemas/config.schema.js';
 import { projectRoot } from '../lib/projectPaths.js';
 import chalk from 'chalk';
 
-export const getEvidenceConfigLegacy = () => {
+/**
+ *
+ * @param {boolean} [silent=false]
+ */
+export const getEvidenceConfigLegacy = (silent) => {
```

Comment:
JSDoc conflicts with function definition. Either add ` = false` to argument, or remove `=false` from JSDoc

```suggestion
/**
 *
 * @param {boolean} [silent]
 */
export const getEvidenceConfigLegacy = (silent) => {
```

```suggestion
/**
 *
 * @param {boolean} [silent=false]
 */
export const getEvidenceConfigLegacy = (silent = false) => {
```